plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemColor.java

```diff
@@ -79,6 +81,13 @@ public ObjectTag getObjectAttribute(Attribute attribute) {
                 }
                 return BukkitColorExtensions.fromColor(pm.getColor()).getObjectAttribute(attribute.fulfill((1)));
```

Comment:
i thought it was correct because i follow on this line.... but, i realize and didn't get how `getObjectAttribute` work. should i change this line too?

Reply:
this whole file is weird and needs cleaning up, but not really required for your PR to fix pre-existing issues (it'd be really handy if you did tho! - check https://github.com/DenizenScript/Denizen/issues/2355 if you're interested in helping cleanup the codebase)